src/h.js

```diff
@@ -1,22 +1,19 @@
-export function h(tag, data) {
-  var node
-  var stack = []
-  var children = []
+var isArray = Array.isArray
```

Comment:
@ngryman Why? It means nothing to uglify. Could this affect your benchmark results?